samples/common/audio_common.cpp

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2018 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "audio_common.h"
+#include <logging_macros.h>
+#include <string>
+
+uint16_t SampleFormatToBpp(oboe::AudioFormat format) {
+    switch (format) {
+      case oboe::AudioFormat::I16:
+        return 16;
+      case oboe::AudioFormat::Float:
+        return 32;
+      default:
+        return 0;
+    }
+}
+
+void PrintAudioStreamInfo(const oboe::AudioStream *stream) {
```

Comment:
This is a useful method for everyone using Oboe. I would consider moving it into `/common/Utilities.cpp` and changing the method signature to: 

    template<>
    const char *convertToText<AudioStream*>(AudioStream *stream)